### PR TITLE
Remove assumptions about git and test

### PR DESCRIPTION
**Pull request by lukebarton**
Perhaps unwelcome (sorry if so) this PR does two things:
- Removes assumption that you're using `git` and want this package to do *anything* with source control.
- Remove assumption that you're using the `test` npm script to run your tests.

It simply removes `yarn`, does an `npm install` and suggests you run your tests and update anything else you might want to do to make sure everything is good.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +27,6 @@ $ yarn global add deyarn
 
 ## Usage
 
-(in the directory of the project to convert, which must be a Git repository)
```
Comment (wearhere) [inline README.md]: This still does have to be done in the directory of the project to convert right? Though it no longer needs to be a Git repository.

File: src/index.js [hunk 2]
```diff
@@ -8,25 +8,7 @@ const editJsonFile = require('edit-json-file');
 
 
 module.exports = () => {
-  console.log(chalk.magenta('\nnpm2yarn.js...\n'));
```
Comment (wearhere) [inline src/index.js]: Oh thanks for removing this, copypasta from our previous npm to yarn script 😆

File: README.md (hunk 1 above)
Comment (lukebarton) [reply README.md]: Done

File: src/index.js (hunk 2 above)
Comment (lukebarton) [reply src/index.js]: np

**Comment by wearhere** [comment]
Thanks! Pinging @spencer-brown to help publish 1.0.2…

**Comment by wearhere** [comment]
Published as 1.0.2! Thanks gain.

**Pull request merged**